java/src/main/java/org/rocksdb/Transaction.java

```diff
@@ -433,6 +433,33 @@ public void rollbackToSavePoint() throws RocksDBException {
    * @param key the key to retrieve the value for.
    * @param exclusive true if the transaction should have exclusive access to
    *     the key, otherwise false for shared access.
+   * @param do_validate true if it should validate the snapshot before doing the read
```

Comment:
1. Could you please rename `do_validate` to `doValidate` so it follows the Java naming conventions.

2. Could you please add the javadoc for the additional parameter `doValidate` on the other methods which are missing it?

3. This occurs in several places, could you correct each please?